modules/gtk/hm.nix

```
@@ -41,6 +41,14 @@ in
   config = lib.mkIf cfg.enable (
     lib.mkMerge [
       {
+        assertions = lib.singleton {
+          assertion = config.gtk.gtk3.extraCss == "" && config.gtk.gtk4.extraCss == "";
+          message = ''
+            Stylix makes `gtk.gtk3.extraCss` and `gtk.gtk4.extraCss` non-functional,
+            use `stylix.targets.gtk.extraCss` instead.
+          '';
```

The error message should be a single line to make it `grep`-able. It should also apply our [message convention](https://github.com/danth/stylix/pull/1155):

```suggestion

          message =
            "stylix: `gtk.gtk3.extraCss` and `gtk.gtk4.extraCss` have no effect. Use `stylix.targets.gtk.extraCss` instead.";
```